### DISCUSSION TITLE
Debugging node.js in code-server

### DISCUSSION THREAD

**Discussion by drmrbrewer** [coder/code-server#4575] [Q&A]
I'm having problems when trying to use the debugging features of code-server.  It just won't seem to take notice of anything I'm specifying in my launch.json.  I figure that I could be launching it incorrectly, but I'm just not sure. 

I've described my woes in a post over on stackoverflow but I'm getting no joy, so I thought I'd try here:

https://stackoverflow.com/q/70086537/4070848

I'm using the latest version of code-server (3.12.0).

**Comment by jsjoeio** [comment]
My guess is it's a code-server issue. Could be this: https://github.com/cdr/code-server/issues/4221

**Reply by drmrbrewer** [reply]
Hmm, OK, and maybe this too: https://github.com/cdr/code-server/issues/4464.  Difficult for me to know what exactly isn't working, and what should happen, because I'm new to code-server so I've never seen it working.

Hopefully working in [code-server v4](https://github.com/cdr/code-server/milestone/32)?

**Reply by jsjoeio** [reply]
🤞 Yes, hopefully! We have a ton of changes from upstream so hopefully it _just works_. 

We're working hard to get v4 out this week so we definitely appreciate the patience! We'll have to test after it's out.
